### PR TITLE
Drop EL8 from Foreman, Katello and Candlepin

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: theforeman.org/pipelines/vars/foreman/nightly.groovy [hunk 1]
```diff
@@ -7,8 +7,7 @@ def foreman_client_distros = [
     'el7'
 ]
 def foreman_el_releases = [
-    'el9',
-    'el8'
```
Comment (evgeni) [inline theforeman.org/pipelines/vars/foreman/nightly.groovy:11]: The pipelines, they are still running

**Review by evgeni — CHANGES_REQUESTED** [review]

File: theforeman.org/pipelines/vars/foreman/nightly.groovy (hunk 1 above)
Comment (evgeni) [reply theforeman.org/pipelines/vars/foreman/nightly.groovy:11]: Sure, but this PR doesn't disable them ;)

Comment (ekohl) [reply theforeman.org/pipelines/vars/foreman/nightly.groovy:11]: Ah, you mean that they should also be dropped from the `centos.org` directory?

Comment (evgeni) [reply theforeman.org/pipelines/vars/foreman/nightly.groovy:11]: At least from `def pipelines_el ` in this file :)

**Comment by Odilhao** [comment]
Done removed the el8 from both install and upgrade pipelines.

**Review by evgeni — APPROVED** [review]

**Pull request merged**